packages/integration/lib/src/widgets/post/comment/comment_list_widget.dart

```diff
@@ -0,0 +1,517 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:likeminds_feed_flutter_core/likeminds_feed_core.dart';
+
+class LMFeedCommentList extends StatefulWidget {
```

Comment:
Should make a style class, or builder delegate for this as mentioned earlier.

Reply:
have added builder in this widget itself to do customisation, it is also customisable from widget_utility class